spring-cloud-contract-stub-runner/src/test/groovy/org/springframework/cloud/contract/stubrunner/StubRunnerExecutorSpec.groovy

```diff
@@ -209,7 +221,9 @@ class StubRunnerExecutorSpec extends Specification {
 
 		@Override
 		<T> void send(T payload, Map<String, Object> headers, String destination, YamlContract contract) {
-			assert !(JsonOutput.toJson(payload).contains("serverValue"))
+			if(payload instanceof String) {
+				assert !(JsonOutput.toJson(payload).contains("serverValue"))
+			}
```

Comment:
maybe we need some other assertion when payload is bytes ?

Reply:
would that require a signature change to that interface to be able to handle for that?

Reply:
Those original assertions just check that something doesn't exist in the payload or headers.